### ISSUE TITLE
Colision on enter building.

### ISSUE THREAD

**Issue by dinhobrj** [Bob74/bob74_ipl#128]
**First of all**
Did you try disabling `bob74_ipl` to see if the issue is still there? Yes
Did you use the latest version of `bob74_ipl` ? Yes
Did you use an up to date server [artifact](https://runtime.fivem.net/artifacts/fivem/build_server_windows/master/) ? Yes

**Describe the bug**
I'm having problems with the resource, when starting it another map of mine collides with the door to enter the building, making it impossible to access, I would like to use bob74_ipl next to this, can anyone help me?

**To Reproduce**
-1302.43,-1348.18,4.52,289.14

**Expected behavior**
Basically it's like I described above, when I start bob74_ipl I can't enter the building.

**Screenshots**
https://cdn.discordapp.com/attachments/1139324514132562021/1176283124007718983/Screenshot_2.png

**Versions**
 - Game build version : Last
 - Resource version : Last
